### PR TITLE
Added an additional capability 'CAPABILITY_AUTO_EXPAND' and updated Documentation

### PR DESCRIPTION
**Pull request by JPurrier**
…he documentation

##### SUMMARY
I added the additional capability  'CAPABILITY_AUTO_EXPAND' I found this was a requirement for the stack i was deploying. So I imagine it will be useful for others. 

##### ISSUE TYPE
- Bugfix Pull Request
The module was missing the ability to add the capability CAPABILITY_AUTO_EXPAND

##### COMPONENT NAME
Small edit to the cloudformation_stack_set module

##### ADDITIONAL INFORMATION
adds CAPABILITY_AUTO_EXPAND to the available options in the list

### CODE REVIEW

File: plugins/modules/cloudformation_stack_set.py [hunk 1]
```diff
@@ -84,7 +84,7 @@
   capabilities:
     description:
     - Capabilities allow stacks to create and modify IAM resources, which may include adding users or roles.
-    - Currently the only available values are 'CAPABILITY_IAM' and 'CAPABILITY_NAMED_IAM'. Either or both may be provided.
+    - Currently the only available values are 'CAPABILITY_IAM' and 'CAPABILITY_NAMED_IAM'. Either or ALL may be provided.
```
Comment (alinabuzachis) [inline plugins/modules/cloudformation_stack_set.py:87]:
```suggestion
    - Currently the only available values are 'CAPABILITY_IAM', 'CAPABILITY_NAMED_IAM' and 'CAPABILITY_AUTO_EXPAND'. Either or ALL may be provided.
```

File: plugins/modules/cloudformation_stack_set.py [hunk 2]
```diff
@@ -84,7 +84,7 @@
...
     - >
         The following resources require that one or both of these parameters is specified: AWS::IAM::AccessKey,
         AWS::IAM::Group, AWS::IAM::InstanceProfile, AWS::IAM::Policy, AWS::IAM::Role, AWS::IAM::User, AWS::IAM::UserToGroupAddition
```
Comment (alinabuzachis) [inline plugins/modules/cloudformation_stack_set.py:90]: Can you also please add a note like:

`Support for C(CAPABILITY_AUTO_EXPAND) was added in release 6.1.0.`

**Review by alinabuzachis — COMMENTED** [review]
@JPurrier Thank you for working on this. Can you also please add a changelog fragment for this change?

**Comment by JPurrier** [comment]
> @JPurrier Thank you for working on this. Can you also please add a changelog fragment for this change?

How do I do this?

**Comment by alinabuzachis** [comment]
> > @JPurrier Thank you for working on this. Can you also please add a changelog fragment for this change?
> 
> @alinabuzachis How do I do this?

@JPurrier I apologise for my late follow up. You should create a changelog fragment using a minor_changes entry https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment. For example https://github.com/ansible-collections/community.aws/blob/main/changelogs/fragments/1557-cloudfront-add-origin-shield.yml

**Pull request closed without merging**